content/blog/elsewhere/yesmore-stop-being-productive.md

```diff
@@ -12,6 +12,7 @@ tags:
   - SuperYesMore
 image:
   src: blog/2017/neck.jpg
+  alt: Mechanical neck drawing
```

Comment:
No idea if there's a better title here. I wasn't sure if this is personal art, or something famous, or ... ?

Reply:
it is personal, and I'm not sure why I chose it for that post. It might be new to oddleventy (not on current oddsite). If it seems random, I'm open to other ideas.

Reply:
I think the image is great but thought it may be a professional piece that I didn't quite remember from 3 years of art history (took too many naps in that class). The image made me stop and read this post while I was working and the message here is really thoughtful.

Reply:
Thanks! That's the cover art from my novel…

Reply:
ah ha! That's why I have seen it before. #famousart